.github/maintainers_guide.md

```diff
@@ -65,6 +65,8 @@ Releasing can feel intimidating at first, but rest assured: if you make a mistak
     -  Push the new tag up to origin: `git push --tags origin`
 5. Distribute the release
     - To publish, you need to be a member of the `slack Org` on npm and set up 2-Factor Auth with your passsword generator of choice. Before you can publish with npm, you must run `npm login` from the command line.
+    - Before publishing a new version, run `rm -rf node_modules/ dist/` to clean your module dependencies in the project first (usually this is not required but in some cases, `npm publish` cannot include all the required files in a package) 
+    - Just in case, run `npm i && npm test && npm pack` and check if the list of the files that will be included in the package are correct
```

Comment:
Perhaps we should be explicit about the minimum required files for a valid npm package?

```suggestion
    - Just in case, run `npm i && npm test && npm pack` and check if the list of the files that will be included in the package contain, at a minimum: `package.json`, `README.md`, `LICENSE`, `CHANGELOG.md`, `dist/index.js`, `dist/App.js`
```